server/routes/api/section.js

```diff
@@ -0,0 +1,159 @@
+const express = require('express');
+const router = express.Router();
+var { Course, SeatingLayout } = require('../../dbSchemas/attendanceSchema');
+
+router.get('/', (req, res) => {
+
+
+});
+
+router.get('/previousSeatingPlans', (req, res) => {
+    // this is the professor ID
+    let professor = req.body.professor;
+
+    SeatingLayout.find({$or : [{default: true} , {created_by : professor}]}, function(err, seatingLayout){
```

Comment:
The seating layouts should return all of them, not only the ones made by the user (current professor)

Reply:
I noticed that too but the way it is done here the parameter of professor is optional so from the frontend I can give nothing as arguments and get back all of the layouts. Because of that, I think it doesn't hurt to have this filter in case we need it.

Reply:
Yeah @JoaoPontes98  and I assumed it made sense for a prof to only access his own classes and the default ones. Obviously if that's no the case then we can remove that, no problem. My only questions is if an individual prof is looking for layouts does it make sense for filter out every other prof's layout?

Reply:
That's a good point Thales, we can put that forward to the client in our next discussion. For now we can do it according to their current specifications.